opentelemetry/src/metrics/meter.rs

```diff
@@ -56,9 +51,9 @@ pub trait MeterProvider {
     /// default name will be used instead.
     fn versioned_meter(
         &self,
-        name: impl Into<Cow<'static, str>>,
```

Comment:
@stormshield-fabs FYI. https://github.com/open-telemetry/opentelemetry-rust/issues/1527 is still to be added for Metrics, which will remove versioned_meter in favor of builder pattern.